### PR TITLE
FI-98: Add issue and PR templates to the repo

### PR DESCRIPTION
**Pull request by okeefm**
This PR adds 3 templates to the repository, through the use of the `.github/` hidden folder:

* `ISSUE_TEMPLATE/bug_report.md`: A template for external users to report suspected bugs through the Inferno "Issues" tab. Selectable by the user if they want to report a bug.
* `ISSUE_TEMPLATE/feature_request.md`: A template for external users to request features in Inferno through the "Issues" tab. Selectable by the user if they want to request a feature.
* `PULL_REQUEST_TEMPLATE/default_pr_template.md`: A Pull Request Checklist, to be filled out by the submitters and both reviewers on every Inferno PR.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/default_pr_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Pull requests into Inferno require the following items to be completed. Submitter and reviewer should check the relevant check boxes when the associated item is done. For items that are not applicable, note it's not applicable ("N/A") and check the box. For example, external Pull Requests do not require ticket links.
```
Comment (Jammjammjamm) [inline .github/PULL_REQUEST_TEMPLATE/default_pr_template.md]: Could you wrap the long lines in this file?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+name: "Bug report"
+about: "Report a bug in Inferno"
+---
+
+Thank you for reporting a possible bug in Inferno! Please fill in as much of the template below as you can.
+
+Subject of the issue
```
Comment (Jammjammjamm) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This file should have some formatting

Comment (okeefm) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It should, in fact. Not sure what bit of copy-pasting I did got rid of it.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+**Steps to reproduce**
+Tell us how to reproduce this issue.
+
+Include:
+
+* The selected test suite
+* The URL of the FHIR server being tested
+* The Test ID, if it's a problem with a specific test.
+* Any required configuration options (client ID, client secret, etc.)
+* Anything else needed to make the issue occur
```
Comment (arscan) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
* Anything else needed to make the issue occur.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+
+**Expected behavior**
+Tell us what should happen
```
Comment (arscan) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Tell us what should happen.
```

File: .github/PULL_REQUEST_TEMPLATE/default_pr_template.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+Pull requests into Inferno require the following items to be completed. Submitter and reviewer 
+should check the relevant check boxes when the associated item is done. For items that are not 
+applicable, note it's not applicable ("N/A") and check the box. For example, external Pull 
+Requests do not require ticket links.
+
+**Submitter:**
+- [ ] This pull request describes why these changes were made.
+- [ ] Internal ticket for this PR:
```
Comment (arscan) [inline .github/PULL_REQUEST_TEMPLATE/default_pr_template.md]:
```suggestion
- [ ] Internal ticket for this PR (core dev team only):
```

File: .github/PULL_REQUEST_TEMPLATE/default_pr_template.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+Pull requests into Inferno require the following items to be completed. Submitter and reviewer 
+should check the relevant check boxes when the associated item is done. For items that are not 
+applicable, note it's not applicable ("N/A") and check the box. For example, external Pull 
+Requests do not require ticket links.
+
+**Submitter:**
+- [ ] This pull request describes why these changes were made.
```
Comment (arscan) [inline .github/PULL_REQUEST_TEMPLATE/default_pr_template.md]:
```suggestion
- [ ] This pull request describes why these changes were made
```

**Pull request merged**